packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1924,6 +1921,22 @@ class StackViewport extends Viewport implements IStackViewport {
     });
   }
 
+  /**
+   * Given the cornerstone image object it will render it in the viewport
+   * This method is used by some utilities, for loading an actual image use
```

Comment:
I do not really understand this sentence. Perhaps it should be broken into two sentences? Maybe a period after `utilities`?